source/common/modules/markdown-editor/hooks/note-preview.js

```diff
@@ -0,0 +1,100 @@
+const tippy = require('tippy.js').default
+const { ipcRenderer } = require('electron')
+
+/**
+ * A hook for displaying link tooltips which can be used to visually
+ * enable users to click a link (without having to press down Ctrl/Cmd)
+ *
+ * @param   {CodeMirror}  cm  The instance to attach to
+ */
+module.exports = (cm) => {
+  cm.getWrapperElement().addEventListener('mousemove', (event) => {
+    let a = event.target
+
+    // Only for note links
+    if (!a.classList.contains('cm-zkn-link')) {
+      return
+    }
+
+    // If there's already a tippy on the instance, don't re-render it
+    if (a.hasOwnProperty('_tippy')) {
+      return
+    }
+
+    // Initialise displayed attributes to 'loading'
+    let title = 'loading'
+    let content = 'loading'
+    let wordCount = 'loading'
+    let time = 'loading'
+    // Create a tippy. This will display the loading values
```

Comment:
The tippy below won't display the loading values, no, it'll just display "Searching For File ...". Did you mean to say "This will contain the values **after they have been loaded**"?

Reply:
Sorry. The intented functionality is to just display "Searching For File...". the fact the values are initialised to 'loading' is a remnant form a previous build plan. I can set them to initialise to 'null'